### ISSUE TITLE
Crash at launch macOS 10.14 Mojave because lib missing

### ISSUE THREAD

**Issue by ferben** [Uncommon/Xit#394]
```
Process:               Xit [17646]
Path:                  /Applications/Xit.app/Contents/MacOS/Xit
Identifier:            com.uncommonplace.Xit
Version:               1.0b11 (1)
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           Xit [17646]
User ID:               502

Date/Time:             2018-12-14 08:49:03.254 +0100
OS Version:            Mac OS X 10.14.2 (18C54)
Report Version:        12
Anonymous UUID:        9E8E5206-32F9-9AC5-C2CA-FDB68FAD6BEB


Time Awake Since Boot: 34000 seconds

System Integrity Protection: enabled

Crashed Thread:        0

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Termination Reason:    DYLD, [0x1] Library missing

Application Specific Information:
dyld: launch, loading dependent libraries

Dyld Error Message:
  Library not loaded: /usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib
  Referenced from: /Applications/Xit.app/Contents/Frameworks/ObjectiveGit.framework/Versions/A/ObjectiveGit
  Reason: image not found
```

This lib is not in default any macOS installation. and should be inside Xit application bundle.
The latest known working version is Xit 1.0.b9.

**Comment by Uncommon** [comment]
Looks like this will be fixed by libgit2/objective-git#668

**Comment by Uncommon** [comment]
The Objective Git fix has been merged in, so this should be okay now.
